spec/Dockerfile.puppetdb

```diff
@@ -0,0 +1,13 @@
+FROM puppet/puppetdb
```

Comment:
what is puppetdb needed for in our workflow?

Reply:
I thought this is where the puppetserver gets the data about tasks from, I may be wrong